### DISCUSSION TITLE
Scenes seem to be corrupted after deploying from backup.

### DISCUSSION THREAD

**Discussion by daCking15** [mozilla/hubs#2908] [General Questions & Discussion] [answered]
![image](https://user-images.githubusercontent.com/15696956/90935431-311ee600-e3d1-11ea-9a75-3de03d24cc69.png)

I'm worried that the scenes have been corrupted after I deployed from a backup. Does anyone have any advice as to how I can save these scenes?

**Comment by emclaren** [comment, marked as answer]
Hi @daCking15  - this isn't an issue we've encountered before, thank you for reporting it. We will need a bit of additional information in order to be able to troubleshoot this and see exactly what's going on

What happens if you try to open one of the scene pages (the pink URL that is made up of random numbers and digits). Does anything load? 

If nothing loads, it would be helpful to get the output from the developer console from this scene page to see if there are any errors there (it looks like you are using Chrome on Windows, so to open them open the Chrome Menu in the upper-right-hand corner of the browser window and select More Tools > Developer Tools.) then select "console" from the top. 

You can copy and paste it with us here or  download it by right clicking on any of the text and press "save as" - its not unusual for there to be a  lot of content shared in the console, but the errors will give us a much better sense of what exactly going wrong
<img width="768" alt="Screen Shot 2020-09-08 at 10 14 07 PM" src="https://user-images.githubusercontent.com/6835324/92810203-32a95180-f372-11ea-9a7a-538b45543080.png">

Are there any errors when you try to go to open them in spoke ( go to your-domain/spoke  to open up your own Spoke instance)

**Reply by robinkwilson** [reply]
Closing this to get it out of our pipeline, please @ me if you need to reopen this.
